public/home/home.js

```diff
@@ -93,11 +87,26 @@ angular.module('pokemon.home', [])
       }
     })
     .catch(function(error) {
-      console.log(error);
+      console.error(error);
     });
   };
-  $scope.userGames();
   
+  userGames();
+
+  pokemonSocket.on('updateAvailGames', function(newGame) {
+    $scope.games = [];
+    $scope.myGames = [];
+    userGames();
+  });
+
+  // test function
+  $scope.changeUserInfo = function() {
+    $window.localStorage.setItem('pokemon.facebookId', "Facebook1234");
+    $window.localStorage.setItem('pokemon.playerName', "Henry");
+    $scope.facebookId = $window.localStorage.getItem('pokemon.facebookId');
+    $scope.playerName = $window.localStorage.getItem('pokemon.playerName');
+    
+  };
```

Comment:
delete this $scope.changeUserInfo function - it was used in testing and is no longer needed